packages/core-transaction-pool/src/memory-transaction.ts

```diff
@@ -1,8 +1,13 @@
 // tslint:disable:variable-name
 
-import { Enums, Interfaces } from "@arkecosystem/crypto";
+import { Interfaces } from "@arkecosystem/crypto";
 import assert from "assert";
 
+/**
+ * A mem pool transaction.
+ * A normal transaction
+ * + a sequence number used to order by insertion time
+ */
 export class MemoryTransaction {
```

Comment:
Rename this to something like `SequentialTransaction` so the comment is not needed to understand what it does, the comment is already superfluous with how simple the code is.

Reply:
Renamed to `SequentialTransaction`, however the comment is still useful because it says what is it being used for, which can't be deduced from `SequentialTransaction`.